Broadcom/symantec-endpoint-protection/_meta/fields.yml

```
@@ -0,0 +1,156 @@
+threat.enrichments:
+  name: threat.enrichments
+  description: ""
+  type: array
+
+threat.enrichments.indicator:
+  name: threat.enrichments.indicator
+  description: ""
+  type: dict
+
+threat.enrichments.indicator.type:
+  name: threat.enrichments.indicator.type
+  description: ""
+  type: text
```

```suggestion
  type: keyword
```
In ECS, this field is a keyword